OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="advance-retract-container">
+
+  <div class="advance-label">
+    ADVANCE
+    <ion-toggle class="advance-toggle" [disabled]="advanceFlag"></ion-toggle>
+  </div>  
+
+  <button class="retract-button" (click)="toggleFlag()"> RETRACT </button>
```

Comment:
So, when "ion-toggle" is **red** for example, the button would be "disabled". To do so, you can look into:
https://www.w3schools.com/tags/att_button_disabled.asp